pillar/devstack.sls

```diff
@@ -6,33 +6,32 @@
 {% set purpose_data =  %}
 {% set purpose_suffix = 'devstack' %}
 
-{% set xqueue_rabbitmq_username = salt.random.get_str(12) %}
+{% set xqueue_rabbitmq_username = 'xqueue_rabbitmq_user' %}
 {% set xqueue_rabbitmq_password = salt.random.get_str(20) %}
-{% set edxapp_rabbitmq_username = salt.random.get_str(12) %}
+{% set edxapp_rabbitmq_username = 'edxapp_rabbitmq_user' %}
 {% set edxapp_rabbitmq_password = salt.random.get_str(20) %}
-{% set admin_mysql_username = salt.random.get_str(12) %}
+{% set admin_mysql_username = 'admin_mysql_user' %}
 {% set admin_mysql_password = salt.random.get_str(20) %}
-{% set xqueue_mysql_username = salt.random.get_str(12) %}
+{% set xqueue_mysql_username = 'xqueue_mysql_user') %}
 {% set xqueue_mysql_password = salt.random.get_str(20) %}
-{% set edxapp_mysql_username = salt.random.get_str(12) %}
+{% set edxapp_mysql_username = 'edxapp_mysql_user' %}
 {% set edxapp_mysql_password = salt.random.get_str(20) %}
-{% set edxapp_mongodb_username = salt.random.get_str(12) %}
+{% set edxapp_mongodb_username = 'edxapp_mongodb_user' %}
 {% set edxapp_mongodb_password = salt.random.get_str(20) %}
-{% set forum_mongodb_username = salt.random.get_str(12) %}
+{% set forum_mongodb_username = 'forum_mongodb_user' %}
 {% set forum_mongodb_password = salt.random.get_str(20) %}
-{% set gitlog_mongodb_username = salt.random.get_str(12)%}
+{% set gitlog_mongodb_username = 'gitlog_mongodb_user')%}
 {% set gitlog_mongodb_password = salt.random.get_str(20) %}
-{% set edxapp_xqueue_username = salt.random.get_str(12) %}
+{% set edxapp_xqueue_username = 'edxapp_xqueue_user' %}
 {% set edxapp_xqueue_password = salt.random.get_str(20) %}
-{% set xqwatcher_xqueue_username = salt.random.get_str(12) %}
+{% set xqwatcher_xqueue_username = 'xqwatcher_xqueue_user' %}
 {% set xqwatcher_xqueue_password = salt.random.get_str(20) %}
 
 {% set CELERY_BROKER_PASSWORD = salt.random.get_str(20) %}
 {% set CELERY_BROKER_USER = 'edxapp' %}
-{% set COMMENTS_SERVICE_KEY =  %}
 {% set DEFAULT_FEEDBACK_EMAIL = 'mitodl-devstack@example.com' %}
 {% set DEFAULT_FROM_EMAIL = 'mitodl-devstack@example.com' %}
-{% set GIT_REPO_DIR = edx.edxapp_git_repo_dir %}
+{% set GIT_REPO_DIR = '/mnt/data/repos' %}
```

Comment:
You mean the vagrant synced folders? That can be anything we specify, but I guess I can change it to /data/repos for clarity